packages/instant/src/components/ui/container.tsx

```diff
@@ -1,3 +1,5 @@
+import { darken } from 'polished';
```

Comment:
this library import syntax is good, and reminds me of our lodash discussion: https://github.com/styled-components/polished#usage

Reply:
yepppp